core/tests/protractor.conf.js

```diff
@@ -319,9 +323,55 @@ exports.config = {
     // will be available. For example, you can add a Jasmine reporter with:
     //     jasmine.getEnv().addReporter(new jasmine.JUnitXmlReporter(
     //         'outputdir/', true, true));
-    var _ADD_SCREENSHOT_REPORTER = true;
 
-    if (_ADD_SCREENSHOT_REPORTER) {
+    var spw = null;
+    var vidPath = '';
+    // Enable ALLVIDEOS if you want success videos to be saved.
+    const ALLVIDEOS = false;
```

Comment:
style nit: can we name this `ALL_VIDEOS` instead?

Reply:
done!